codegen.yml

```diff
@@ -15,7 +15,7 @@ generates:
     preset: near-operation-file
     presetConfig:
       extension: .generated.ts
-      baseTypesPath: ./common/src/types/schema.ts
+      baseTypesPath: ~@openmsupply-client/common
```

Comment:
The path for the base paths (i.e. schema.ts) becomes relative in the operation fils, which typescript doesn't like as it's not within the dir specified. This fixes that.. though it also uses this same import path for within `common`.. which seems to work ok right now, but it doesn't seem right. However, it's better than the original problem so..